mozilla/blurts-server
# MNTOR-3246- Add error state and limitations apply page

# References:

Jira: MNTOR-3246




# Screenshots

Error state:

<img width="988" alt="image" src="https://github.com/mozilla/blurts-server/assets/13066134/a37b4ba9-446b-4284-856d-b20898724235">



# How to test (if applicable)

Follow the same flags/env setup as https://github.com/mozilla/blurts-server/pull/4635.

Selecting the "Limited time, restrictions apply" link should take you to `/limitations-apply` in a new tab:

<img width="1723" alt="image" src="https://github.com/mozilla/blurts-server/assets/13066134/aa68cd34-3465-4dd6-9a08-1be49d9c9bba">


# Checklist (Definition of Done)
